packages/@storefront/flux-capacitor/CHANGELOG.md

```diff
@@ -4,6 +4,70 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [minor]
+### Changed
+- SF-1256: Persist sort and past purchase sort labels in the application store.
```

Comment:
We should consider standardizing our tenses in the changelog. Changelogs are traditionally in the past tense. I'm not going to ask you to change this, if you don't want to, though.

Reply:
Agreed, will change.